ignite/handlers/checkpoint.py

```diff
@@ -377,13 +385,17 @@ def load_objects(to_load: Mapping, checkpoint: Mapping, **kwargs) -> None:
             # single object and checkpoint is directly a state_dict
             key, obj = list(to_load.items())[0]
             if key not in checkpoint:
+                if isinstance(obj, (nn.DataParallel, nn.parallel.DistributedDataParallel)):
```

Comment:
That's just a comment but  is not useless if user puts `obj.module` in mapping. What happens if someone really wants to save DDP ? 
Note that `type` methods discussed on slack ignite channel should work ;)

Reply:
Well, with actual behaviour it is not possible. But if we are using `auto_model` it is not garanteed that model is DDP, so user needs to check the type before calling `.module`.

> Note that type methods discussed on slack ignite channel should work ;)

I do not see how it could work. Anyway, using `type` to create new classes is also rather obscure way of patching things :)

Reply:
> I do not see how it could work. Anyway, using `type` to create new classes is also rather obscure way of patching things :)

override `load_dict_state` to return module one

Reply:
Let's avoid such monkey-patching :)